<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>468</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Hello Word</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>res/icons/main.png</normaloff>res/icons/main.png</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_word_name">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>WordName</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_pronunciation">
        <property name="font">
         <font>
          <family>Lingoes Unicode</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Pronunciation</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_button_uk_pron">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>res/icons/uk.png</normaloff>res/icons/uk.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="push_button_us_pron">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>res/icons/us.png</normaloff>res/icons/us.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="text_browser_definition">
      <property name="font">
       <font>
        <family>华文中宋</family>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>470</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件 (&amp;F)</string>
    </property>
    <addaction name="action_choose_lexicon"/>
    <addaction name="action_stat"/>
    <addaction name="action_new_mode"/>
    <addaction name="action_review_mode"/>
    <addaction name="action_settings"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>帮助 (&amp;H)</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolbar">
   <property name="windowTitle">
    <string>快捷栏</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_choose_lexicon"/>
   <addaction name="action_stat"/>
   <addaction name="action_new_mode"/>
   <addaction name="action_review_mode"/>
   <addaction name="action_settings"/>
   <addaction name="separator"/>
   <addaction name="action_help"/>
   <addaction name="action_about"/>
  </widget>
  <action name="action_choose_lexicon">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/lexicon.png</normaloff>res/icons/lexicon.png</iconset>
   </property>
   <property name="text">
    <string>选择词库 (&amp;L)</string>
   </property>
   <property name="statusTip">
    <string>单词将从您选择的词库中抽取</string>
   </property>
  </action>
  <action name="action_stat">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/stat.png</normaloff>res/icons/stat.png</iconset>
   </property>
   <property name="text">
    <string>词汇统计 (&amp;S)</string>
   </property>
   <property name="statusTip">
    <string>词汇统计</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/about.png</normaloff>res/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>关于 (&amp;A)</string>
   </property>
   <property name="statusTip">
    <string>显示版本和作者信息</string>
   </property>
  </action>
  <action name="action_review_mode">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/review.png</normaloff>res/icons/review.png</iconset>
   </property>
   <property name="text">
    <string>复习 (&amp;R)</string>
   </property>
   <property name="statusTip">
    <string>复习</string>
   </property>
  </action>
  <action name="action_settings">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/config.png</normaloff>res/icons/config.png</iconset>
   </property>
   <property name="text">
    <string>设置 (&amp;O)</string>
   </property>
   <property name="statusTip">
    <string>设置</string>
   </property>
  </action>
  <action name="action_new_mode">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/new.png</normaloff>res/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>学习 (&amp;N)</string>
   </property>
   <property name="statusTip">
    <string>学习</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/exit.png</normaloff>res/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>退出 (&amp;X)</string>
   </property>
   <property name="statusTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_help">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/help.png</normaloff>res/icons/help.png</iconset>
   </property>
   <property name="text">
    <string>按键说明 (&amp;B)</string>
   </property>
   <property name="statusTip">
    <string>按键说明</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
